components/SortableTable/THead.vue

```diff
@@ -150,9 +150,16 @@ export default {
 <style lang="scss" scoped>
   .sortable > A {
     display: inline-block;
+    outline: 0;
```

Comment:
When you click on tab-index-able things like links they get an outline.  This shouldn't be one weird exception to that.

Reply:
pushing change to fix